c2corg_ui/static/js/documentediting.js

```diff
@@ -265,10 +267,9 @@ app.DocumentEditingController.prototype.submitForm = function(isValid) {
  */
 app.DocumentEditingController.prototype.successSave_ = function(response) {
   // redirects to the document view page
-  var url = '/{module}/{id}/{culture}'
-      .replace('{module}', this.module_)
-      .replace('{id}', response['data']['document_id'])
-      .replace('{culture}', this.culture_);
+  goog.asserts.assert(this.culture_ !== null);
+  var url = app.utils.buildDocumentUrl(
```

Comment:
There are many possible URLs for a document. What about renaming to `buildGetDocumentUrl`?